common/types.go

```diff
@@ -224,21 +224,23 @@ const (
 type Layer string
 
 const (
-	LayerConsensus Layer = "consensus"
-	LayerEmerald   Layer = "emerald"
-	LayerCipher    Layer = "cipher"
-	LayerSapphire  Layer = "sapphire"
-	LayerPontusx   Layer = "pontusx"
+	LayerConsensus  Layer = "consensus"
+	LayerEmerald    Layer = "emerald"
+	LayerCipher     Layer = "cipher"
+	LayerSapphire   Layer = "sapphire"
+	LayerPontusx    Layer = "pontusx"
+	LayerPontusxDev Layer = "pontusx_dev"
```

Comment:
is it weird that we use underscore here but no underscore in the routes?

Reply:
hm we could change the routes to have the underscore but it seems a bit strange. we do have to keep the underscore here though to align with the client-sdk config.